es5.md

```diff
@@ -531,7 +531,7 @@ A unary operator token.
 interface UpdateExpression <: Expression {
     type: "UpdateExpression";
     operator: UpdateOperator;
-    argument: Expression;
+    argument: Pattern | CallExpression;
```

Comment:
Ah, I didn't know about that early error in ES5. Okay, then I am fine with this as long as we add an item to the "philopsophy" section of the README that states we do not wish to represent programs that would otherwise be syntactically valid if not for their early errors.